Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ruby:2.5.1-alpine
+MAINTAINER apply for legal aid team
+
+RUN apk --update add --virtual build-dependencies \
+                                    build-base \
+                                    libxml2-dev \
+                                    libxslt-dev \
+#                                    postgresql-dev \
```

Comment:
It is my opinion that we shouldn't commit commented-out lines, however, on the assumption that we're going to use this in the near future, it can stay.

Reply:
I revised my opinion after I saw that @adrianclay thought the same. Removed.